site/content/3.13/data-science/graphrag/services/importer.md

```diff
@@ -279,14 +222,127 @@ to send an input file to the Importer service:
 
 ## Verifying the import
 
-You can verify that the import was successful by checking your ArangoDB database:
+You can verify the state of the import process via the following endpoint:
+
+```
+GET /gen-ai/v1/project_by_name/<your_project>
+```
+
+For example, the `status` object found within `importerServices` may contain the following
+properties:
+
+```json
+"status": {
+    "status": "service_completed",
+    "progress": 100,
+    "message": ""
```

Comment:
@hkernbach is this message supposed to be an empty string?

Reply:
Right now this is valid. We will improve this in the future. Right now this message can be used optionally. e.g. in case an error occurred, details will be in the message property.